sdk/healthinsights/azure-health-insights-cancerprofiling/src/samples/README.md

```diff
@@ -0,0 +1,57 @@
+---
+page_type: sample
+languages:
+  - java
+products:
+  - azure
+  - azure-cognitive-services
+  - azure-health-insights
+urlFragment: healthinsights-java-samples
+---
+
+# Azure Health Insights client library samples for Java
+
+Azure Health Insights Cancer Profiling samples are a set of self-contained Java programs that demonstrate interacting with Azure self-contained service using the client library. Each sample focuses on a specific scenario and can be executed independently.
+
+## Key concepts
+
+<!-- Key concepts are explained in detail [here][SDK_README_KEY_CONCEPTS]. -->
+Key concepts are explained in detail [here]
+## Getting started
+
+<!-- Getting started explained in detail [here][SDK_README_GETTING_STARTED]. -->
+Getting started explained in detail [here]
+
+## Examples
+
+The following sections provide code samples covering common scenario operations with the Azure Health Insights CancerProfiling client library.
+
+All of these samples need the endpoint to your Health Insights resource, and your Health Insights API key.
+
+|**File Name**|**Description**|
+|----------------|-------------|
+|[SampleInferCancerProfile.java][infer_cancer_profiling]|Get the inferred cancer staging|
```

Comment:
suggest to put the link of the file together with the file name.

Reply:
sample files are currently not in main.
My plan is to update the links after the PR is merged.
Unless there is a process of creating links to a files that are not in main